worldedit-core/src/main/resources/lang/strings.json

```diff
@@ -69,7 +69,17 @@
     "worldedit.gmask.disabled": "Global mask disabled.",
     "worldedit.gmask.set": "Global mask set.",
     "worldedit.toggleplace.pos1": "Now placing at pos #1.",
+    "worldedit.toggleplace.pos1-offset": "Now placing at an offset of ({0}, {1}, {2}) from pos #1.",
     "worldedit.toggleplace.player": "Now placing at the block you stand in.",
+    "worldedit.toggleplace.player-offset": "Now placing at an offset of ({0}, {1}, {2}) from the block you stand in.",
+    "worldedit.toggleplace.min": "Now placing at the minimum of the current selection.",
+    "worldedit.toggleplace.min-offset": "Now placing at an offset of ({0}, {1}, {2}) from the minimum of the current selection.",
+    "worldedit.toggleplace.max": "Now placing at the maximum of the current selection.",
+    "worldedit.toggleplace.max-offset": "Now placing at an offset of ({0}, {1}, {2}) from the maximum of the current selection.",
+    "worldedit.toggleplace.world": "Now placing at world origin.",
+    "worldedit.toggleplace.world-offset": "Now placing at ({0}, {1}, {2}).",
+    "worldedit.toggleplace.here": "Now placing at the block you stand in *right now*.",
+    "worldedit.toggleplace.here-offset": "Now placing at an offset of ({0}, {1}, {2}) from the block you stand in *right now*.",
```

Comment:
this is actually a lot worse than my last nitpick on language. I genuinely find some of these confusing as to whether they are static locations or dynamic given the use of qualifiers like "current", "right now", or none.

i think part of this is that all of the absolute(/static) locations use different terminology when they realistically can all just tell the player the absolute coordinates being used as the placement position. that is, even though `world(-offset)` and `here-(offset)` might parse and calculate differently, they are all static coordinate locations (as far as i can tell), so that should really be all the player cares about. i wouldn't even bother telling them the offset. in all four cases just give them the absolute x/y/z position.
now, all the relative/dynamic locations can contrast that by giving the reference point (player, min/max, etc) and optional offset. i think that would make these a lot more clear.

Reply:
`worldedit.toggleplace.here[-offset]` aren't used in practice, since the special case HERE is replaced by WORLD before it ever hits the `printInfo` line in `placementImpl`.
These strings are here, just in case someone _else_ needs to use `printInfo` on the result of `PlacementType.HERE.getTranslationKey[Offset]` for some reason.

Reply:
In truth, I didn't think it's worth making a new converter that accepts either `PlacementType` and the string "here".
That's a lot of extra code for just a tiny gain in code quality.
Making `setPlacement` reject it is a good idea though. I'll get on it.
I'll make `getTranslationKey[Offset]` throw as well while I'm at it.

Reply:
since those are gone now, I'm gonna resolve the conversation.